dejure.org.js

```diff
@@ -0,0 +1,242 @@
+	"label": "dejure.org",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://dejure\\.org/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-01-16 20:00:51"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/dienste/vernetzung/')>-1) {
+		if (getSearchResults(doc, true)) {
+			return "multiple";
+		} else {
+			return "case";
+		}
+	} else if (url.indexOf('/gesetze/')>-1 && url.indexOf('.html')>-1) {
+		return "statute";
+	}
+}
+
+
+//We only handle the cases that there are multiple cases on the
+//same pages in multiples.
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var itemsDoc = [];
+	var found = 0;
+	var rows = ZU.xpath(doc, '//div[contains(@class, "rspr_inhalt")]');
+	for (var i=0; i<rows.length; i++) {
+		var title = ZU.xpathText(rows[i], './/td[contains(@class, "urteilszeile")]');
+		if (!title) continue;
+		found++;
+		items[i] = title;
+		itemsDoc[i] = rows[i];
+	}
+	if (checkOnly) {
+		if (found>1) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	return found ? [items, itemsDoc] : false;
+}
+
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if (type == "multiple") {
+		var results = getSearchResults(doc, false);
+		Zotero.selectItems(results[0], function (items) {
+			if (!items) {
+				return true;
+			}
+			for (var i in items) {
+				scrape(results[1][i], url, "case");
+			}
+		});
+	} else {
+		scrape(doc, url, type);
+	}
+}
+
+
+function scrape(doc, url, type) {
+	var item = new Zotero.Item(type);
+	
+	if (type == "case") {
+		var headline = ZU.xpathText(doc, './/td[contains(@class, "urteilszeile")]');
+		var posComma = headline.indexOf(",");
+		var posDash = headline.indexOf("-");
+		if (posComma>0) {
+			item.court = headline.substr(0, posComma);
+		}
+		if (posDash > 0) {
+			item.docketNumber = headline.substr(posDash+2);
+		}
+		item.dateDecided = ZU.strToISO(headline);
+		item.title = headline;
+		
+		var previousDecisions = ZU.xpath(doc, './/div[h4[contains(., "Verfahrensgang")]]/ul/li');
+		item.history = previousDecisions.map(function(li) { return li.textContent; } ).join("; ");
```

Comment:
I changed the arrow notation here, because JSHint warns me:
```
'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').
```
Now, it also works in browser without ES6 support.